cinema/urls.py

```diff
@@ -15,7 +15,20 @@
 router.register("cinema_halls", CinemaHallViewSet)
 router.register("movies", MovieViewSet)
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
 
 urlpatterns = [path("", include(router.urls))]
 
 app_name = "cinema"
+
+{"tickets": [
+    {
+        "movie_session": 7,
+        "seat": 1,
+        "row": 1
+    },
+    {
+        "movie_session": 8,
+        "seat": 2,
+        "row": 1
+    }]}
```

Comment:
There is a JSON-like structure at the end of the file, which seems to be misplaced. This might be leftover test data or an accidental inclusion. It should be removed to avoid any potential issues.